src/app/layout.tsx

```diff
@@ -6,21 +6,18 @@ export const metadata: Metadata = {
   description: "모델과 디자이너 매칭 플랫폼",
 };
 
-export default function RootLayout({
+const RootLayout = ({
   children,
 }: Readonly<{
   children: React.ReactNode;
-}>) {
+}>) => {
   return (
-    <html lang="ko">
-      <head>
-        <meta charSet="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
-        <link rel="icon" href="/images/icon.svg" type="image/svg+xml" />
-      </head>
-      <body>
+    <html lang="ko" suppressHydrationWarning>
+      <body className="min-h-screen overflow-x-hidden bg-white">
```

Comment:
```suggestion
    <html lang="ko" suppressHydrationWarning>
      <body className="bg-white">
        <div className="w-[375px] mx-auto min-h-screen overflow-x-hidden shadow-2xl">
          {children}
        </div>
      </body>
    </html>
```
레이아웃 자체를 375px, mx-auto(좌우 정렬) 로 해두는게 좋을거 같습니다
추가로
```
// login/page.tsx - w-[375px] mx-auto 제거
<div className="relative bg-white font-sans">

// SignupFunnel.tsx - w-[375px] mx-auto 제거  
<div className="bg-white min-h-screen relative">
```
위 사항 반영한다면 singup이랑 login 페이지 최상단 div도 수정해야될거 같습니당

Reply:
위 코드는 shadow-2xl 추가한 겁니다!
<img width="1920" height="988" alt="image" src="https://github.com/user-attachments/assets/09c6407f-829f-46ef-923a-7ae17af0fa98" />